roles-in-tech/security.md

```diff
@@ -68,33 +89,26 @@ These are some misconceptions that people have in their mind but it is not alway
 
-
-OWASP releases a regular list of the [top 10 most critical web application security risks](https://www.owasp.org/index.php/Category:OWASP_Top_Ten_Project). Here are the 2017 highlights:
-- Injection: validate everything before you give it to an interpreter! Here we focus on JavaScript sanitization.
-- Broken Authentication and Session Management: practice this by using the above libraries. Better than passwords, try SAML.
-- XSS and CRSF: XSS is basically injection in the DOM, and you can't prevent CRSF until you've tackled all the XSS bugs. Try it at home with [Google](https://xss-game.appspot.com/) and [Excess XSS](http://excess-xss.com/).
-- Encryption: sensitive data should be encrypted in transit and at rest. Also, you are not a mathematician; never try to roll your own encryption.
-
 ### Guided Practice
-
-- Apprentice will use popular libraries to add authentication and session management to the Pinterest clone project. Perhaps using Google or [OpenID](http://docs.identityserver.io/en/release/quickstarts/7_javascript_client.html#refjavascriptquickstart) for SAML auth is better than trying Passport.
-  - Also consider [Auth0](/oauth/o-auth.md).
-- Next, you can validate user input with Parsley and validate fields with Validator.
-- Use DOMPurify to prevent XSS.
+- Talk to as many professionals as you can [National Cyber Security Centre](https://live.ncsc.gov.uk/topics/new-talent) is worth for who lives in UK
 
 ### Independent Practice
+- *You can play a game* called [PicoCTF](https://picoctf.com/about) that involves using reverse engineering, hacking, decryption and other skills to resolve the mystery behind the sudden disappearance of your character’s father.
 
-Build a page template to make it easier to prevent CSRF.
-
-### Challenge
+- *Experiment with second computer*. Firstly, install Linux, BSD, Qubes OS and other operating systems on it.
+Then, plug it to an Arduino board (i.e. an electronics platform that can read input like a light or touch sensor or your social media messages), and turn it into an output (publishing something on Twitter, switching on an LED or whatever).
+Don’t forget. Install security and (with permission) monitor the networks that you connect to.
 
-If you are feeling inclined, or interested in red teaming, you can experiment with [Insecure Labs](http://www.insecurelabs.org) or a [Kali Linux](https://www.kali.org/) VM and read about its rich [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) tool suite.
 
 ### Check for Understanding
-
-The next step is to try to XSS each other's sites.
+- You can start playing [CTFs](https://ctftime.org/ctf-wtf/). It is a great way to learn ethical hacking.
+- Learn how to code! - [codecademy](https://www.codecademy.com/about),
```

Comment:
Line 105 and 106 are a little too general - could you remove them or replace them with something about role sin security?